src/helpers/misc/delete_stage_instance.ts

```diff
@@ -0,0 +1,11 @@
+import { rest } from "../../rest/rest.ts";
+import { endpoints } from "../../util/constants.ts";
+
+/** Deletes the Stage instance. Requires the user to be a moderator of the Stage channel. */
+export async function deleteStageInstance(channelId: bigint) {
+  // deno-lint-ignore ban-types
+  return await rest.runMethod<{}>(
```

Comment:
If the body is not dispatched from the server, the runMethod() returns an empty JSON-parsed object instead.
The type could be changed to `Record<...>` instead, however, it was obscure and ambiguous, hence, `{}`.